californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfig.java

```diff
@@ -413,6 +415,8 @@ public NetworkConfig() {
 	public NetworkConfig(NetworkConfig config) {
 		this.properties = new Properties();
 		this.properties.putAll(config.properties);
+		this.observers.addAll(config.observers);
```

Comment:
In the meantime, the `NetworkConfig` turns more into immutable one, so the observer are more to be removed.